src/main/java/com/xpanxion/java/assignments/student7/Worker7.java

```diff
@@ -60,4 +59,17 @@ public void ex4() {
         System.out.println(formatter.format(foodTotals));
     }
 
+    public void ex5() {
+    var people = DataAccess.getPeople();
+
+    var fixedSsn = people.stream()
+            .filter(p -> p.getId()<=3)
+            .map(p -> {
+                var split = p.getSsn().split("-");
+                p.setSsn(split[2]);
+                return p;
+            })
+            .collect(Collectors.toList());
```

Comment:
FYI - Instead of `.collect(Collectors.toList());` we can simply use `.toList();`